Net/SmartIRC.php

```diff
@@ -2501,16 +2500,26 @@ protected function _callTimeHandlers()
             if ($microtimestamp >= $handlerinfo['lastmicrotimestamp']
                 + ($handlerinfo['interval'] / 1000.0)
             ) {
-                $methodobject = &$handlerinfo['object'];
-                $method = $handlerinfo['method'];
+                $callback = &$handlerinfo['callback'];
```

Comment:
Can you make a quick commit removing the `&` operator? That way I don't have to merge, then pull and re-merge locally and all that.
As soon as I see it, I'll merge the PR.